taichi/cache/gfx/cache_manager.h

```diff
@@ -37,7 +37,8 @@ class CacheManager {
 
   explicit CacheManager(Params &&init_params);
 
-  CompiledKernelData load_or_compile(CompileConfig *config, Kernel *kernel);
+  CompiledKernelData load_or_compile(const CompileConfig *config,
```

Comment:
Why is the argument sometimes `CompileConfig *` and sometimes `CompileConfig &`? Can we unify them to `CompileConfig &`?

Reply:
History reason lol.
I will fix all `const CompileConfig *` to `const CompileConfig &` in future, which is part of #7002.

![image](https://user-images.githubusercontent.com/52557189/212595332-1e9ba957-1a55-4640-b089-f8b1d981e002.png)